ios/Podfile.lock

```diff
@@ -587,4 +587,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: fe5332bfe94340e09768442eda194c2d51dd1b1f
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.9.1
```

Comment:
@andreaforni any idea why this snuck in?

Reply:
Thanks @ShaunBaker, I've committed it by error. Let me restore it.